### DISCUSSION TITLE
Adding feature to share quotation

### DISCUSSION THREAD

**Discussion by rosdyana** [novalagung/muslimboard#73] [Ideas]
Dear @novalagung, I am so glad that I found this browser extension. Jazakallah Khairan Katsiran.
Btw, I have a request to share the quote from the Quran verse on social media platforms.
![image](https://user-images.githubusercontent.com/4516635/195035320-2a291647-70e0-4baa-b6a3-f8ac92037ab7.png)

**Comment by novalagung** [comment]
@rosdyana really happy to hear that!

feel free to share anything from this browser extension to any social media :D
you can also contribute some quotes.

regarding the ability to share it on socials, we used to have that on the earlier version, but I can't remember why I removed it.
let me try to check about the feature this weekend.

**Comment by ekypradhana** [comment]
Hi @novalagung , hope you are doing well!

lately sometimes I screenshot the quote and use it as a WhatsApp story, I imagine if there is a way to export it to a 'story-fit' dimension, it'd be great! haha :D

but never mind as long as the aim is fulfilled, thank you for the great work!
hope the jariyah comes to you and your family insyaAllah.

![story](https://github.com/novalagung/muslimboard/assets/11922274/34055cb7-70f3-4889-a677-e8ab5e86ea37)
